static/js/angular/rules.js

```diff
@@ -12,19 +12,19 @@ btvRulesApp.controller("RulesCtrl", function($scope) {
         },
         {
             rule: "No spoilers in Mane Chat",
-            description: "We try and keep our chat room spoiler-free as much as possible. So if a clip or episode is released/leaked before a Saturday Premiere, we would prefer if you take all spoiler discussion to #SpoilerWagonBTV until AFTER the new ep airs on The HUB."
+            description: "We try and keep our chat room spoiler-free as much as possible. So if a clip or episode is released/leaked before it's premiere (usually on Saturdays), we would prefer if you take all spoiler discussion to #SpoilerWagonBTV until one day AFTER the new episode/movie originally airs on Discovery Family."
         },
```

Comment:
Also what is the HUB?


On Tue, May 10, 2016 at 1:21 AM, Gavin Lambert <notifications@github.com>
wrote:

> In static/js/angular/rules.js
> <https://github.com/BronyTV/bronytv.net/pull/43#discussion_r62613806>:
>
> > @@ -12,19 +12,19 @@ btvRulesApp.controller("RulesCtrl", function($scope) {
> >          },
> >          {
> >              rule: "No spoilers in Mane Chat",
> > -            description: "We try and keep our chat room spoiler-free as much as possible. So if a clip or episode is released/leaked before a Saturday Premiere, we would prefer if you take all spoiler discussion to #SpoilerWagonBTV until AFTER the new ep airs on The HUB."
> > +            description: "We try and keep our chat room spoiler-free as much as possible. So if a clip or episode is released/leaked before it's premiere (usually on Saturdays), we would prefer if you take all spoiler discussion to #SpoilerWagonBTV until one day AFTER the new episode/movie originally airs on Discovery Family."
> >          },
>
> before it's premiere should be before its premiere.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/BronyTV/bronytv.net/pull/43/files/810d915409e63eb2ae23e0b70343020db8f47572#r62613806>
>